clio-server/src/test/scala/org/broadinstitute/clio/server/dataaccess/ReadGroupElasticsearchDAOSpec.scala

```diff
@@ -20,7 +20,7 @@ class ReadGroupElasticsearchDAOSpec
   }
 
   it should "updateReadGroupMetadata" in {
-    val key = ModelReadGroupKey("barcodeURGM1", 2, "library3")
+    val key = ModelReadGroupKey("barcodeURGM1", 2, "library3", "GCP")
```

Comment:
My 2¢:

Main code avoiding _Magic_ is definitely good S.O.P.

I've seen a really good blog post that argued that even tests _should_ use constants and not magic values. I can't find it at the moment. But a) I haven't worked on a team project yet where someone else has introduced this principle consistently, and b) when I played with test-constant-everything myself, I didn't like how verbose I felt the repetitive test code looked. That said, up to you and your tech-lead/P.O./tie-breaker-reviewer.